.ci/Jenkinsfile

```diff
@@ -78,8 +78,8 @@ pipeline {
             steps {
               withGithubNotify(context: "Test-${PLATFORM}") {
                 dir("${BASE_DIR}"){
-                  withGoEnv(){
-                    goTestJUnit(options: '-v ./...', output: 'junit-report.xml')
+                  withMageEnv(){
+                    cmd(label: 'Go unitTest', script: 'mage unitTest')
                   }
```

Comment:
Not familiar with the `withMageEnv` call but I assume it generates the `junit-report.xml` ?